invokeai/app/invocations/latent.py

```diff
@@ -198,7 +206,7 @@ def get_conditioning_data(
             # for ddim scheduler
             eta=0.0,  # ddim_eta
             # for ancestral and sde schedulers
-            generator=torch.Generator(device=unet.device).manual_seed(0),
+            generator=torch.Generator(device=unet.device).manual_seed(seed),
```

Comment:
@keturn `LatentsField` now optionally includes the `seed` used for noise generation, so we get deterministic-but-not-same-y generations on SDE & Ancestral samplers

Reply:
hmm. If `seed` here is the same as was used for the generator that generated the initial noise, is this going to re-create that initial noise its first time through the scheduler? That's probably not what we want.

In which case... well, the simplest thing would be to throw in some static adjustment here, like
```py
SCHEDULER_SALT = 2023080917  # some constant we make up, with enough bits to be interesting
scheduler_seed = initial_seed ^ SCHEDULER_SALT
ConditioningData(generator=Generator().seed(scheduler_seed))
```

another alternative might be to mirror the diffusers API and pass around the Generator itself instead of seeds... but the Generator is stateful so if the order that nodes execute in is not stable, that would prevent things from being reproducible.

Reply:
Added xor by 0xFFFFFFFF to flip all bits.